ui/apps/platform/src/Containers/Vulnerabilities/NodeCves/Overview/NodeCvesOverviewPage.tsx

```diff
@@ -177,6 +182,35 @@ function NodeCvesOverviewPage() {
                     </FlexItem>
                 </Flex>
             </PageSection>
+            <PageSection variant="light">
+                <Alert
+                    isInline
+                    variant="info"
+                    title="Results may include Node CVEs obtained from Scanner V4."
+                    component="p"
+                >
+                    <Flex
+                        direction={{ default: 'column' }}
+                        spaceItems={{ default: 'spaceItemsMd' }}
+                    >
+                        <Flex direction={{ default: 'row' }}>
```

Comment:
If there is not any additional content planned for this alert, I think both of these `<Flex>` wrappers can be removed without any ill effect.

Reply:
Let's see how it looks like - changed in [c2ac4b8](https://github.com/stackrox/stackrox/pull/14130/commits/c2ac4b87c08f6db892aaf060f8f5b7e232ec1567)